### DISCUSSION TITLE
Feature request - hotkeys for stay on top and sizes

### DISCUSSION THREAD

**Discussion by Arwenz47** [smplayer-dev/smplayer#791] [Ideas]
I think SMplayer is a great player and I enjoy it very much!

I would like to place a feature request: implement a default hotkey for "stay on top", a hotkey for "video size 50%" and a hotkey for switching to next and previous video size (50%, 75%, 100%, 125%, 150%).

Thank your very much!

**Comment by LandisTwo** [comment]
assuming Linux, 
these are functions of your window manager.
I've assigned the 'hot key' of Alt+U (keep above) and Alt+P to 'pin' / un-pin (all desktops) and Meta+H ('hide')
resizing can be done in 'window rules' (KDE)
and re-sizing to the resolution of each video is an option in SMPlayer > Preferences > Interface > Main > Autoresize: [whenever it's needed] (select from dropdown)
- and maybe select 'Remember position and size' too
